scripts/lib/workspace_command.py

```
@@ -0,0 +1,778 @@
+    validate_invocation(command, require_required=not is_dry_run)
+    if is_dry_run:
+        return 0
+    return run(command)
+
+
+def run(command: Command) -> int:
+    env = command_env(command)
+    cwd = command_cwd(command)
+    if command.path.suffix == ".py":
+        return run_python(command, env, cwd)
+    return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)